src/redux/explorer.js

```diff
@@ -237,6 +264,47 @@ export const explorerInit = () => async (dispatch, getState) => {
   }
 };
 
+const toAssetSubscriptionPayload = tokensArray => {
```

Comment:
if we do comment above, then we can get rid of this function and just pass in the `newAssetCodes` to the `assetPricesSubscription` directly